### PR TITLE
GEODE-4794 User Guide - update description of `gfsh configure pdx` command.

### PR DESCRIPTION
**Pull request by davebarnes97**

### CODE REVIEW

File: geode-docs/tools_modules/gfsh/command-pages/configure.html.md.erb [hunk 1]
```diff
@@ -27,30 +27,31 @@ Configure Portable Data eXchange for all the cache(s) in the cluster.
...
 
 <a id="topic_jdkdiqbgphqh__section_C27BE964CE554180A65968DBEBF50B23"></a>
-Configures <%=vars.product_name%>'s Portable Data eXchange for all the cache(s) in the cluster. This command does not effect on the running members in the system. This command persists the pdx configuration in the locator with cluster configuration service.
 
-**Note:**
-This command should be issued before starting any data members.
+Configures <%=vars.product_name%>'s Portable Data eXchange for all the cache(s) in the cluster
+and persists the pdx configuration in the locator with the cluster configuration service.
+
+This command does not affect the running members in the system, so should be issued before starting any data members.
```
Comment (joeymcallister) [inline geode-docs/tools_modules/gfsh/command-pages/configure.html.md.erb]: I think there's a word missing here. Maybe either "so it should be issued" or "and so should be issued."

File: geode-docs/tools_modules/gfsh/command-pages/configure.html.md.erb [hunk 2]
```diff
@@ -27,32 +27,31 @@ Configure Portable Data eXchange for all the cache(s) in the cluster.
...
-| <span class="keyword parmname">\\-\\-portable-auto-serializable-classes</span> | Configures ReflectionBasedAutoSerializer as the PDX serializer for member classes. The patterns that are matched against domain class names to determine whether they should be serialized. Serialization done by the PDX autoserializer will throw an exception if the object of these classes are not portable to non-Java languages (equivalent to `check-portability=true`).                                                                                                                                       | none    |
+**Parameters, configure pdx:**
 
-<span class="tablecap">Table 1. Configure PDX Parameters</span>
+| Name                                       | Description | Default |
+|--------------------------------------------|-------------|---------|
```
Comment (jinmeiliao) [inline geode-docs/tools_modules/gfsh/command-pages/configure.html.md.erb]: If this is copied from the command help, then I am fine with it. If not, then we will need to change the code to match this.

Comment (jinmeiliao) [reply geode-docs/tools_modules/gfsh/command-pages/configure.html.md.erb]: I don't have much experience on what `configure PDX` do. The actual help text probably needs to be looked by the person in the storage?

Comment (davebarnes97) [reply geode-docs/tools_modules/gfsh/command-pages/configure.html.md.erb]: Don't worry, no code changes needed - earlier work on this JIRA ticket took care of that.
This PR addresses some of the wording, just trying to make it more precise.

**Pull request merged**